feature/checker-inbox-task/src/main/java/com/sparklead/feature/checker_inbox_task/checker_inbox_and_task/presentation/CheckerInboxTasksScreen.kt

```diff
@@ -0,0 +1,210 @@
+    val state = checkerInboxTasksViewModel.checkerInboxTasksUiState.collectAsState().value
+    val isRefreshing by checkerInboxTasksViewModel.isRefreshing.collectAsState()
+    val swipeRefreshState = rememberSwipeRefreshState(isRefreshing = isRefreshing)
+
+    LaunchedEffect(key1 = true) {
+        checkerInboxTasksViewModel.loadCheckerTasksBadges()
+    }
+
+
+    Scaffold(
+        modifier = Modifier.fillMaxSize(),
+        containerColor = White,
+        topBar = {
+            TopAppBar(
+                colors = TopAppBarDefaults.mediumTopAppBarColors(containerColor = White),
+                navigationIcon = {
+                    IconButton(
+                        onClick = { onBackPressed() },
+                    ) {
+                        Icon(
+                            imageVector = Icons.Rounded.ArrowBackIosNew,
+                            contentDescription = null,
+                            tint = Black,
+                        )
+                    }
+                },
+                title = {
+                    Text(
+                        text = stringResource(id = R.string.checker_inbox_and_pending_tasks),
+                        style = TextStyle(
+                            fontSize = 22.sp,
+                            fontWeight = FontWeight.Medium,
+                            fontStyle = FontStyle.Normal,
+                            fontFamily = FontFamily(Font(R.font.outfit_medium))
+                        ),
+                        color = Black,
+                        textAlign = TextAlign.Start
+                    )
+                },
+            )
+        }
+    ) { padding ->
+        SwipeRefresh(
+            state = swipeRefreshState,
+            onRefresh = { checkerInboxTasksViewModel.loadCheckerTasksBadges() }
+        ) {
+            when (state) {
+                is CheckerInboxTasksUiState.Error -> {
+                    MifosSweetError(message = "Failed to Load Check Inbox") {
+                        checkerInboxTasksViewModel.loadCheckerTasksBadges()
+                    }
+                }
+
+                is CheckerInboxTasksUiState.Loading -> {
+                    MifosCircularProgress()
+                }
+
+                is CheckerInboxTasksUiState.Success -> {
+                    Column(modifier = Modifier.padding(padding)) {
+                        TaskOptions(
+                            leadingIcon = R.drawable.ic_mail_outline_24dp,
+                            option = "Checker Inbox",
+                            badge = state.checkerInboxBadge
+                        ) {
+                            checkerInbox()
+                        }
+                        TaskOptions(
+                            leadingIcon = R.drawable.ic_supervisor_account_24dp,
+                            option = "Client Approval",
+                            badge = "0"
+                        ) {
+
```

Comment:
why onclick is missing here?

Reply:
Till now there is no next screen for that feature.